src/task_1.3/task1_es6.js

```diff
@@ -0,0 +1,25 @@
+import { createInterface } from 'readline';
+
+const rl = createInterface({
+  input: process.stdin,
+  output: process.stdout,
+  terminal: false
+});
+
+const reverseInput = () => {
```

Comment:
Consider refactoring these functions to avoid recursion.
Additionally, consider always extracting callbacks into separate functions to improve readability.